protected/humhub/modules/admin/models/forms/AuthenticationSettingsForm.php

```diff
@@ -23,6 +23,9 @@ class AuthenticationSettingsForm extends \yii\base\Model
     public $internalRequireApprovalAfterRegistration;
     public $internalUsersCanInvite;
     public $defaultUserGroup;
+    public $MinUserNameLength;
```

Comment:
Please use stick to our coding standards.

http://docs.humhub.org/dev-core-coding-standards.html
https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#1-overview